<launch>
    <!-- Argument fÃ¼r die Liste der Roboter -->
    <arg name="robot_names" default="[]" />
    <arg name="UR_prefixes" default="['UR10_l', 'UR10_r']" />

    <node pkg="handling" type="update_all_relative_poses.py" name="update_all_relative_poses" output="screen">
        <rosparam param="robot_names" subst_value="True">$(arg robot_names) </rosparam>
        <rosparam param="UR_prefixes" subst_value="True">$(arg UR_prefixes) </rosparam>
    </node>
</launch>